packages/ckeditor5-dev-env/lib/release-tools/changelog/templates/commit.hbs

```diff
@@ -55,6 +55,11 @@
       {{~this.owner}}/
     {{~/if}}
     {{~this.repository}}#{{this.issue}}
-  {{~/if}}{{/each}}
+  {{~/if}}{{/each}}.
+{{~/if}}
+{{#if body}}
+
+
+{{body}}
```

Comment:
Those 2 empty lines have to be here. The first one inserts a new line at the end of commit sentence, the second one separates the sentence and additional description.